pysatNASA/instruments/methods/cdaweb.py

```diff
@@ -964,9 +964,12 @@ def cdas_list_remote_files(tag='', inst_id='', start=None, stop=None,
     elif start == stop:
         stop = start + dt.timedelta(days=1)
 
-    if isinstance(start, pds._libs.tslibs.timestamps.Timestamp):
-        start = start.tz_localize('utc')
-        stop = stop.tz_localize('utc')
+    # Ensure that valid date types are used.
+    start = pysat.utils.time.filter_datetime_input(start)
+    stop = pysat.utils.time.filter_datetime_input(stop)
+
+    # CDAS WS needs a time for the stop date.
```

Comment:
What does CDAS WS stand for?

Reply:
I should change this to cdasws, which is the package name.